src/network/network_models/powermodels_interface.jl

```diff
@@ -294,24 +294,35 @@ end
 
 #### PM accessor functions ########
 
+function PMvarmap(system_formulation::Type{S}) where {S <: PM.DCPlosslessForm}
+    pm_var_map = Dict{Type,Dict{Symbol, Union{Symbol,NamedTuple}}}()
+
+    pm_var_map[PSY.Bus] = Dict(:va => :theta)
+    pm_var_map[PSY.ACBranch] = Dict(:p => (from_to = :Pbr_fwd, to_from = nothing))
+    pm_var_map[PSY.DCBranch] = Dict(:p_dc => (from_to = :Pbr_fwd, to_from = nothing))
```

Comment:
We should discriminate between DC and AC branches.

Reply:
These get concatenated with the branch type so it will end up being `:Pbr_fwd_HVDCLine` or `:Pbr_fwd_Line`. I think that further AC/DC distinction isn't needed.